query/stdlib/influxdata/influxdb/from.go

```diff
@@ -649,6 +651,9 @@ func (FromKeysRule) Rewrite(keysNode plan.Node) (plan.Node, bool, error) {
 }
 
 func createFromSource(prSpec plan.ProcedureSpec, dsid execute.DatasetID, a execute.Administration) (execute.Source, error) {
+	span, ctx := tracing.StartSpanFromContext(context.TODO())
```

Comment:
Are you sure about this being a TODO? it seems that context.Background() would work better here.

Reply:
Just a note for the query team. I figure they're neck-deep in refactors, so this is enough of a billboard.